src/main/java/hudson/plugins/accurev/CheckForChanges.java

```diff
@@ -295,4 +192,42 @@ public static boolean checkStreamForChanges(AccurevServer server,
         }
         return isTransLatestThanBuild;
     }
+
+    private static boolean changesMatchFilter(TaskListener listener, Collection<String> serverPaths, List<String> filters) {
```

Comment:
I used it during development to print some debug info. It can indeed be removed.